compiler/src/dotty/tools/dotc/sbt/ExtractDependencies.scala

```diff
@@ -91,15 +92,16 @@ class ExtractDependencies extends Phase {
       } finally pw.close()
     }
 
-    if (ctx.sbtCallback != null) {
-      collector.usedNames.foreach {
-        case (clazz, usedNames) =>
-          val className = classNameAsString(clazz)
-          usedNames.names.foreach {
-            case (usedName, scopes) =>
-              ctx.sbtCallback.usedName(className, usedName.toString, scopes)
-          }
-      }
+    if (ctx.incrementalEnabled) {
```

Comment:
I don't think that check is needed since isRunnable already prevents running the phase if incrementalEnabled is false

Reply:
good catch

Reply:
gone now